src/cap-description.adoc

```diff
@@ -92,37 +92,77 @@ Therefore, it is only possible to encode a subset of all combinations.
-| 0b1111   | ✔ | ✔ | ✔ | ✔ | ✔
+| Encoding    | R | W | C | X | ASR | Mode | Notes
+8+| *Quadrant 0: Non-capability data read/write*
+| 0b00_000   |   |   |   |   |     | N/A | No permissions
+| 0b00_001   | ✔ |   |   |   |     | N/A | Data RO
+| 0b00_010   7+| reserved
+| 0b00_011   7+| reserved
+| 0b00_100   |   | ✔ |   |   |     | N/A | Data WO
+| 0b00_101   | ✔ | ✔ |   |   |     | N/A | Data RW
+| 0b00_110   7+| reserved
+| 0b00_111   7+| reserved
+8+| *Quadrant 1: Executable capabilities*
+| 0b01_000   | ✔ | ✔ | ✔ | ✔ |  ✔  | Legacy  | Execute + ASR (see <<infinite-cap>>)
+| 0b01_001   | ✔ | ✔ | ✔ | ✔ |  ✔  | Purecap | Execute + ASR (see <<infinite-cap>>)
+| 0b01_010   | ✔ |   | ✔ | ✔ |     | Legacy  | Execute + Data & Cap RO
+| 0b01_011   | ✔ |   | ✔ | ✔ |     | Purecap | Execute + Data & Cap RO
+| 0b01_100   | ✔ | ✔ | ✔ | ✔ |     | Legacy  | Execute + Data & Cap RW
+| 0b01_101   | ✔ | ✔ | ✔ | ✔ |     | Purecap | Execute + Data & Cap RW
+| 0b01_110   | ✔ | ✔ |   | ✔ |     | Legacy  | Execute + Data RW
+| 0b01_111   | ✔ | ✔ |   | ✔ |     | Purecap | Execute + Data RW
+8+| *Quadrant 2: Reserved*
+| 0b10_000   7+| reserved
+| 0b10_001   7+| reserved
+| 0b10_010   7+| reserved
+| 0b10_011   7+| reserved
+| 0b10_100   7+| reserved
+| 0b10_101   7+| reserved
+| 0b10_110   7+| reserved
+| 0b10_111   7+| reserved
+8+| *Quadrant 3: Capability data read/write*
+| 0b11_000   7+| reserved
+| 0b11_001   7+| reserved
+| 0b11_010   7+| reserved
+| 0b11_011   | ✔ |   | ✔ |   |     | N/A | Data & Cap RO
+| 0b11_100   7+| reserved
+| 0b11_101   7+| reserved
+| 0b11_110   7+| reserved
+| 0b11_111   | ✔ | ✔ | ✔ |   |     | N/A | Data & Cap RW
 |==============================================================================
 
+
 The encoding in xref:cap_perms_encoding32[xrefstyle=short] is chosen to
 facilitate hardware implementations. Therefore, it can be worked out if the
-permissions are granted as follows:
+permissions are granted as follows, assuming that all reserved cases are ignored:
+
+* <<c_perm>>: `P[4] OR (P[3] AND P[1:0] != 0b11)`
+* <<w_perm>>: `P[2] OR P[4:1] == 0b0100`
+* <<r_perm>>: `P[4:0] != 0b00000 AND P[4:0] != 0b00100`
+* <<x_perm>>: `P[4:3] == 0b01`
+* <<asr_perm>>: `P[4:1] == 0b0100`
 
-* <<c_perm>>: bit 0 is set
-* <<w_perm>>: bit 1 is set
-* <<r_perm>>: bits 3 or 2 are set
-* <<x_perm>>: bit 3 is set
-* <<asr_perm>>: bits 3 and 2 are set
+The mapping of bit values to decoded permissons in <<cap_perms_encoding32>> is based on the following encoding strategy:
+[#cap_perms_encoding_description32,width="100%",options=header,cols="^2,^1,^1,^1,6",align="center"]
+|==============================================================================
+| Bit 4&3 | Bit 2    | Bit 1      |  Bit 0     | Notes
+5+<|_Data & capability memory access (Quandrant 2+3)_
+|   Q=0b11 | Write 2+| Reserved=0b11          <| Reserved bits for future extensions must be 1
+|   Q=0b01 3+| Reserved=0b111                 <| Reserved bits for future extensions must be 1
```

Comment:
Is this correct? The subtitle says quadrants 2+3 but Q=0b11 and Q=0b01 (i.e. 3 and 1)